pkg/platform/api/settings.go

```diff
@@ -109,6 +118,24 @@ var urlsByService = map[Service]*url.URL{
 		Host:   constants.DefaultAPIHost,
 		Path:   constants.HasuraInventoryAPIPath,
 	},
+	ServiceUpdateInfo: {
+		Scheme: "https",
+		Host:   constants.DefaultAPIHost,
+		Path:   constants.UpdateInfoAPIPath,
+	},
+}
+
+var configuredAPIHost string
+
+func RegisterConfigListener(cfg *config.Instance) error {
+	configMediator.AddListener(constants.APIHostConfig, func() {
+		configuredAPIHost = cfg.GetString(constants.APIHostConfig)
+	})
+	return nil
+}
```

Comment:
```suggestion
func RegisterConfigListener(cfg *config.Instance) {
	configMediator.AddListener(constants.APIHostConfig, func() {
		configuredAPIHost = cfg.GetString(constants.APIHostConfig)
	})
}
```

There's no error to handle.